BLSQ/iaso
# IA-3619: Go from the Submissions screen on the web to the Request Change screen related to this Submission

Explain what problem this PR is resolving
- Go from the Submissions screen on the web to the Request Change screen related to this Submission
Related JIRA tickets : [IA-3619](https://bluesquare.atlassian.net/browse/IA-3619)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Added a WidgetPaper containing the changeRequest related to the current instance
- Added change_requests key in as_full_model dict

## How to test
- Create a change request with a an reference_instance
- Open the reference instance details and check if the create change requests part is there

## Print screen / video

[Screencast from 2024-11-11 09-43-45.webm](https://github.com/user-attachments/assets/5c42bd9f-1eef-4583-adbe-2d953e865a06)

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3619]: https://bluesquare.atlassian.net/browse/IA-3619?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
